XtermBlazor/OnKeyEventArgs.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System;
+using Microsoft.AspNetCore.Components.Web;
+
+namespace XtermBlazor
+{
+    /// <summary>
+    /// OnKey event data
+    /// </summary>
+    public class OnKeyEvent
```

Comment:
Use `KeyEventArgs` instead of `OnKeyEvent`, should be look like this
```cs
public class KeyEventArgs : EventArgs
```

Similar to
https://github.com/BattlefieldDuck/XtermBlazor/blob/main/XtermBlazor/RenderEventArgs.cs